### PR TITLE
Add cluster turnup

### PR DESCRIPTION
**Pull request by camh-**
Add k3s cluster turnup Makefile, scripts and manifests.

This starts a k3s single node master running sealed secrets,
cert-manager, metallb and traefik. The default k3s traefik installation
is not used as that is version 1.X and I want 2.X so we install it
ourselves from a yaml manifest.

The default installation is incomplete as it needs some site-specific
settings. The final commit adds that for my (camh) site - two domains,
my email address and my LAN IP range. This can be used as a template for
a different site deployment.

It is expected that this will be expanded to deploy non-master nodes
later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+* jq
+
+Install these with your OS package manager.
+
+### k3s binary installation
+
+This repo has a `Makefile` and download script to download and install
+k3s, instead of using the k3s-provided method of curl-downloading a
+shell script and piping it into `sh` as root. It is not as flexible as
+the k3s installation script, but I don't need it to be
```
Comment (juliaogris) [inline README.md]: something missing here at end of line? `.` or `:` maybe?
or maybe `. Install with`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+or
+
+    make install-k3s install-k3s-links install-systemd install-dirs
+
+To undo all that `make install` does, run
+
+    make uninstall
+
+This will delete any data in any local persistent volumes that
```
Comment (juliaogris) [inline README.md]: I'd add a Warning or similar prefix.

**Review by juliaogris — COMMENTED** [review]
ok, i will just post these comments here before they go out of date (or maybe they have with the updated commit message).
apologies - feel free to ignore all. continuing now.

File: README.md (hunk 1 above)
Comment (camh-) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (camh-) [reply README.md]: Done

**Review by juliaogris — APPROVED** [review]
🍵

**Pull request merged**